src/docker-osx-dev

```diff
@@ -942,9 +1014,7 @@ function configure_includes {
 #
 function watch_and_sync {
   log_info "Starting docker-osx-dev file syncing"
-  configure_boot2docker_ssh_key
-  init_boot2docker
-  install_rsync_on_boot2docker
+  init_docker_host
```

Comment:
I can move back here. But I believe this slows down the startup of `docker-osx-dev` after the install.